doc/kubernetes/modules/ROOT/partials/running/infinispan-credentials.adoc

```diff
@@ -0,0 +1,37 @@
+. Configure the credential to access the Infinispan cluster.
++
+Keycloak needs this credential to be able to access the cluster.
+By default, the Infinispan Operator generates credentials with a random password and stores it in a secret.
```

Comment:
I think we should just document how to define a pre-configured password and then provide a link to the Infinispan Operator docs as further reading at the end of this section. I don't think a Keycloak user cares about Infinispan's defaults etc, just what they need to do to get started and using pre-configured credentials is the easiest way.